tck-dist/src/main/asciidoc/concurrency-tck-reference-guide.adoc

```diff
@@ -0,0 +1,625 @@
+= Jakarta Concurrency TCK Reference Guide
+:author: Kyle Aure
+:email: kyle.aure@ibm.com
+:revnumber: v1.0
+:toc:
+:sectnums:
+
+:APIShortName: Concurrency
+:APIShortNameLC: concurrency
+:APILongName: Jakarta Concurrency
+:APIVersion: 3.0.0
+:APISpecSite: https://jakarta.ee/specifications/concurrency/3.0/
+:APIEclipseSite: https://projects.eclipse.org/projects/ee4j.cu
+:APIGitSite: https://github.com/jakartaee/concurrency
+:TCKTestPlatform: TestNG
+:RuntimeTestCount: 164
+:SigPluginGAV: org.netbeans.tools:sigtest-maven-plugin:1.6
+
+== Preface
+
+This guide describes how to download, install, configure, and run the Technology Compatibility Kit (TCK) used to verify the compatibility of an implementation of the {APILongName} specification.  
+
+The specification describes the job specification language, Java programming model, and runtime environment for {APILongName} applications.
+
+=== Licensing
+
+The {APILongName} TCK is provided under the *Eclipse Foundation Technology Compatibility Kit License - v 1.0* [https://www.eclipse.org/legal/tck.php].
+
+=== Who Should Use This Guide
+
+This guide will assist in running the test suite verifies implementation compatibility for:
```

Comment:
Add the word "which" ?
`This guide will assist in running the test suite, which verifies implementation compatibility for:`